.markdownlint.jsonc

```diff
@@ -0,0 +1,33 @@
+// Configuration for markdownlint
+// See https://github.com/DavidAnson/markdownlint/blob/v0.32.1/schema/.markdownlint.jsonc
+{
+  
+  // Default state for all rules
+  "default": true,
+
+  // MD013/line-length : Line length : https://github.com/DavidAnson/markdownlint/blob/v0.32.1/doc/md013.md
+  // Justification: Line length standard has not been defined
+  "MD013": false,
+
+  // MD024/no-duplicate-heading : Multiple headings with the same content : https://github.com/DavidAnson/markdownlint/blob/v0.32.1/doc/md024.md
+  // Justification: Some sections have a common template sub-heading format.
+  "MD024": {
+    // Only check sibling headings
+    "siblings_only": true
+  },
+
+  // MD029/ol-prefix : Ordered list item prefix : https://github.com/DavidAnson/markdownlint/blob/v0.32.1/doc/md029.md
+  // Justification: When lists have indented content, it resets the one style lists, and the numbering must be manually set, but then one style can be used from then on.
+  "MD029": false,
+
+  // MD033/no-inline-html : Inline HTML : https://github.com/DavidAnson/markdownlint/blob/v0.32.1/doc/md033.md
+  "MD033": false,
+
+  // MD041/first-line-heading/first-line-h1 : First line in a file should be a top-level heading : https://github.com/DavidAnson/markdownlint/blob/v0.32.1/doc/md041.md
+  "MD041": false,
```

Comment:
We use H1 in all our markdown files, don't we? Maybe leave this on default too?

Reply:
Removed for flexibility in case we used any fragment markdown files, but I'll enable as suggested.